style2.css

```diff
@@ -0,0 +1,128 @@
+*{
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.title {
+  padding: 2%;
+  text-align: center;
+}
+
+#check-menu {
+  display: none;
+}
+
+.menu {
+  background: #b39ddb;
+  display: block;
+  float: right;
+  height: 100vh;
+  width: 30%;
+}
+
+.menu__link {
+  display: block;
+  width: 100%;
+  padding: 5%;
+  background-color: #b39ddb;
+  color: white;
+  text-decoration: none;
+  border-bottom: .2px solid #e0e0e0;
+  text-align: center;
+  font-size: 1.2em;
+}
+
+.menu__link:hover {
+  background: grey;
+}
+
+        /* media */
```

Comment:
Codacy found an issue: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/css17__/pullRequest?prid=5991486)